tests/xtimer_msg/main.c

```diff
@@ -38,13 +38,14 @@ struct timer_msg {
     msg_t msg;
 };
 
-struct timer_msg msg_a = { .interval = 2*(1000000), .text = "Hello World", };
-struct timer_msg msg_b = { .interval = 5*(1000000), .text = "This is a Test" };
+struct timer_msg msg_a = { .interval = (2 * US_PER_SEC),
+                           .text = "Hello World", };
```

Comment:
Very minor, there's a trailing comma before `}`. I could be removed to be consistent the other struct (`timer_msg msg_b`).

Reply:
oops missed that, will change in a second 😄